kitsune/wiki/urls.py

```diff
@@ -11,6 +11,7 @@
     re_path(r"^/revision/(?P<revision_id>\d+)$", views.revision, name="wiki.revision"),
     re_path(r"^/history$", views.document_revisions, name="wiki.document_revisions"),
     re_path(r"^/edit$", views.edit_document, name="wiki.edit_document"),
+    re_path(r"^/edit/document$", views.edit_document_doc, name="wiki.edit_document_doc"),
```

Comment:
What do you think of the following instead?
```suggestion
    re_path(r"^/edit/metadata$", views.edit_document_metadata, name="wiki.edit_document_metadata"),
```